test/torchaudio_unittest/datasets/gtzan_test.py

```diff
@@ -65,9 +64,7 @@ def test_training(self):
             n_ite += 1
         assert n_ite == len(self.training)
 
-    def test_validation(self):
-        dataset = gtzan.GTZAN(self.root_dir, subset='validation')
-
+    def test_validation(self, dataset):
```

Comment:
As it is, this function raises an error in Circle CI since it is detected as a test. It should simply be called `_test_validation`.